CITATION.cff

```diff
@@ -1,6 +1,27 @@
 # YAML 1.2
 ---
+title: "pyMOR"
 authors: 
+  - family-names: Fritze
+    given-names: "René"
```

Comment:
Are quotes necessary here? Should my last name also have them?

Reply:
I think the inconsistencies stem from me starting from a generator website thingy back when, and then sometimes dropping the quotes when editing and sometimes not. 
To make the document valid, I think we only need quotes around numeric-looking values.

Reply:
The [validator](https://github.com/pymor/pymor/runs/5228483066?check_suite_focus=true) seems happy w/o the quotes.